model/src/main/kotlin/utils/SortedSetConverters.kt

```diff
@@ -77,7 +77,7 @@ class ScanResultSortedSetConverter : StdConverter<Set<ScanResult>, Set<ScanResul
                     this += it.provenance.sourceArtifact.url
                 }
                 else -> {
-                    // Cannot happen as ScanResults in ScannerRun
+                    // Do not append anything, because unknown provenance does not have any properties.
```

Comment:
Nit: "append" is the verb used for `buildString`, but for `buildList` it'd be "add".